src/components/RefundButton.tsx

```diff
@@ -234,6 +226,68 @@ const RefundButton = (props: {
         return transactionToRefund;
     });
 
+    const message = () =>
+        props.buttonOverride ??
+        (lockupTransaction.state == "errored"
+            ? t("no_lockup_transaction")
+            : valid() || !refundAddress() || !props.swap()
+              ? t("refund")
+              : t("invalid_address", { asset: props.swap()?.assetSend }));
```

Comment:
Let's not nullish coalesce a double ternary. Now that we have a function for it, we should make it readable